static/app/components/group/groupSummaryWithAutofix.tsx

```diff
@@ -125,7 +213,7 @@ function AutofixSummary({
   const insightCards: InsightCardObject[] = [
     {
       id: 'root_cause_description',
-      title: t('Root cause'),
+      title: t('Root Cause'),
```

Comment:
Should update the issue summary cards to have the same case as well? I originally did it this way to align w/ that